pkg/metrics/metrics.go

```diff
@@ -56,7 +57,20 @@ func userIsOptedIn(app *application.Avalanche) bool {
 	return app.Conf.ConfigFileExists() && app.Conf.GetConfigBoolValue(constants.ConfigMetricsEnabledKey)
 }
 
-func HandleTracking(cmd *cobra.Command, commandPath string, app *application.Avalanche, flags map[string]string) {
+func HandleTracking(
+	app *application.Avalanche,
+	flags map[string]string,
+	err error,
+) {
+	if sent {
```

Comment:
if user calls command 1, sent will be set to true after tracking info is sent. What about when user calls command 2, since sent would have been set to true here, will the tracking info be sent?

Reply:
I guess best way to test this is see if we can see the second command on the dashboard after first command is called